sphinx/templates/latex/latex.tex_t

```diff
@@ -47,11 +47,15 @@
 \release{<%= release %>}
 \author{<%= author %>}
 \newcommand{\sphinxlogo}{<%= logo %>}
-\renewcommand{\releasename}{<%= releasename %>}
+<%- if releasename or release %>
+\renewcommand{\releasename}{<%= releasename or _('Release') %>}
+<%- else %>
+\renewcommand{\releasename}{}
+<%- endif %>
 <%= makeindex %>
 <%= body %>
 <%= atendofbody %>
 <%= indices %>
-\renewcommand{\indexname}{<%= indexname %>}
+\renewcommand{\indexname}{<%= _('Index') %>}
```

Comment:
Should this be defined in document environment?

Reply:
not sure what you mean, which part should be defined there? It can't be moved before `<%= indices %>` because they have their own `\renewcommand\indexname`.  In writers/latex.py:

https://github.com/sphinx-doc/sphinx/blob/c659a562915d9771e3c246a3cade2ea9079ca10d/sphinx/writers/latex.py#L862-L863

(Python Module Index e.g.). Or perhaps you mean `sphinx.writers.generate_indices()` should incorporate this line as last line of `<%= indices =>`? But then if user uses custom template with general index located before domain indices, there is problem.

Reply:
I thought preamble is better. But it's my wrong. Thank you for explanation.